### DISCUSSION TITLE
HugeRTE is up at npm and composer (packagist)

### DISCUSSION THREAD

**Discussion by carlosmintfan** [hugerte/hugerte#25] [Announcements]
HugeRTE is now up at [npm](https://npmjs.com/package/hugerte) and [Packagist](https://packagist.org/packages/hugerte/hugerte). **IMPORTANT:** If you have installed HugeRTE 1.0.0 via npm, you must urgently upgrade to a newer version because it [contained the completely wrong source code instead of the right built code](https://github.com/hugerte/hugerte/issues/23).

**Comment by markeaston63** [comment]
Thanks! I was wondering what was going on. I have just been quiet as I have been gettng our new NavEditor implemented using TinyMCE 6.8.3. All sorts of features in there. Been working weekends as well - so no spare time. I have another week or two until it is completed and pased by QA. We will release in November. Once I come up for a breather I will attend to matters here! Im still here.

**Comment by markeaston63** [comment]
PS it says in the ReadMe Visit the HugeRTE Website. When you click on that it takes you to https://www.tiny.cloud/.

**Reply by carlosmintfan** [reply]
Thanks! Will update it soon.

**Comment by rxXxc** [comment]
Hi,

I want to use in VueComponent where earlier I was having below code to use tinymce editor. What is the best way to achieve the same editor in VueComponent as I used in tinymce earlier.

import Editor from '@tinymce/tinymce-vue'

Also the package version is "@tinymce/tinymce-vue": "^5.1.0",

Please suggest the replacement in these case.

thanks in advance!

**Reply by carlosmintfan** [reply]
Discussion in #31
